### ISSUE TITLE
Running MATSim MaaS from command line

### ISSUE THREAD

**Issue by MaroFlan** [matsim-org/matsim-code-examples#705]
Hi, my research requires me to use the dvrp contrib of MATSim from the command line, but I can't seem to get it to work. I am still new to MATSim so please forgive my ignorance. 
I tried to use the command "java -Xmx512m -cp /path/to/matsim.jar org.matsim.run.Controler /path/to/config.xml" with the appropriate paths as stated in the user guide, but I am getting the error "java.lang.ClassNotFoundException: org.matsim.run.Controler". I am using the snapshot version of matsim-maas. Do I need to run a different main class? Clearly I am missing something.

Thanks for your help.

**Comment by Janekdererste** [comment]
We need a little more information. Could you please provide the following things:
- Which example did you try to run (A link to a repository would be most helpful)?
- Please supply the entire error message
- If you have altered any input files, please provide those files as well

**Comment by MaroFlan** [comment]
Thanks for your answer,

I tried to run the mielec_2014_02 scenario from https://github.com/matsim-org/matsim-maas with the mielec_taxi_config_assignment.xml config. I didn't alter any of the files used, as I just wanted to test starting the simulation from the command line.
I used the command "java -Xmx512m -cp .\target\matsim-maas-master-SNAPSHOT.jar org.matsim.run.Controler .\scenarios\mielec_2014_02\mielec_taxi_config_assignment.xml" from the main directory "matsim-maas-master" and got the error:
Error: Could not find or load main class org.matsim.run.Controler
Caused by: java.lang.ClassNotFoundException: org.matsim.run.Controler

So my guess is that I need to run a different class, but I don't know which one.
